CHANGELOG.md

```diff
@@ -8,18 +8,41 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - Support for `idempotencyKey` for `POST` methods by @aydrian & @rileylnapier
+- Support for [Lists API](https://docs.courier.com/reference/lists-api) by @aydrian
+  - `POST /send/list` via `client.sendListOrPattern(params, config)`
+  - `GET /profiles/{recipient_id}/lists` via `client.profiles.getRecipientLists(recipientId)`
+  - `GET /lists` via `client.lists.getLists(params)`
+  - `GET /lists/{list_id}` via `client.lists.getList(listId)`
+  - `PUT /lists/{list_id}` via `client.lists.replaceList(listId, {...})`
```

Comment:
one this one, we are wondering if this should be:
-  `client.lists.createOrReplace`

...or should we have convenience methods like:

- `PUT /lists/{list_id}` via `client.lists.create`
- `PUT /lists/{list_id}` via `client.lists.update` || `PUT /lists/{list_id}` via `client.lists.replace` (we like `update` more and that's likely more consistent with apis like [stripe](https://stripe.com/docs/api/radar/value_lists/update))

that way we have a path to introduce a `POST` if we wanted to do so. thoughts?

Reply:
I was following the standard we've used elsewhere. Posts are creates or merge and Puts are replace even if they also create. The only place we use create is in brands because it doesn't merge.